ztools/x509/testdata/name.constraint.test.cert

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Try the top or the bottom, whatever works. Example: https://cs.chromium.org/chromium/src/net/data/ssl/certificates/self-signed-invalid-sig.pem?q=self-signe&sq=package:chromium&l=1